e2e/cmd/e2e.go

```diff
@@ -186,6 +186,7 @@ func main() {
 		"latest_single_instance": func(s *scenario.Scenario) {},
 		"latest_but_one_single_instance": func(s *scenario.Scenario) {
 			s.KSMVersion = "v1.9.7"
```

Comment:
shouldn't be 1.9.8?

Reply:
No, as it tests last but one KSM version.